common/src/main/kotlin/entity/Interactions.kt

```diff
@@ -395,3 +399,50 @@ sealed class InteractionResponseType(val type: Int) {
 
     }
 }
+
+
+@KordPreview
+@Serializable
+data class DiscordApplicationCommandPermissions(
```

Comment:
This should be named `DiscordGuildApplicationCommandPermissions`?

Reply:
I guess it only works for guilds

Reply:
a GuildApplicationCommandPermissions contains ApplicationCommandPermissions
see: https://discord.com/developers/docs/interactions/slash-commands#guildapplicationcommandpermissions